cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -375,12 +381,12 @@ public void upgrade() {
     }
 
     public static void main(final String[] _args) throws Exception {
-//        Logger l = Logger.getLogger(Channel.class.getName());
-//        l.setLevel(ALL);
-//        ConsoleHandler h = new ConsoleHandler();
-//        h.setLevel(ALL);
-//        l.addHandler(h);
-//
+        Logger l = Logger.getLogger(ROOT_LOGGER_NAME);
+        l.setLevel(SEVERE);
+        ConsoleHandler h = new ConsoleHandler();
+        h.setLevel(SEVERE);
+        l.addHandler(h);
```

Comment:
I suppose we should merge this fix first anyway even if it is a soon-to-be-dead PR